packages/test_track/lib/src/domain/get_visitor_config.dart

```diff
@@ -33,12 +34,20 @@ class GetVisitorConfig {
     required String visitorId,
     required AppVersionBuild appVersionBuild,
   }) async {
-    final response = await _client.get<Map<String, dynamic>>(
-      '/api/v4/apps/${appVersionBuild.appName}/versions/${appVersionBuild.version}/builds/${appVersionBuild.buildTimestamp}/visitors/$visitorId/config',
+    final appVisitorConfig = await _client.execute<Json, AppVisitorConfig>(
+      IdempotentGetRequest(
```

Comment:
i see that this adds the `isIdempotentOptionsKey: true` to the request--I presume this key is something consumed by this visitor config endpoint and does something on the backend?

Reply:
That key is just used by the SDK to indicate requests that are idempotent (i.e can be invoked > 1 times without risk of producing incorrect state). It's used by the `RetryInterceptor` so it knows whether or not it can retry requests! I didn't bother documenting it primarily because it's not exported and only used internal to the SDK, but LMK if you think that's a bad idea 🙃

Reply:
cool! i think it's fine as is

Reply:
I documented it, since I think this convo is proof that it's not 100% apparent at the outset! So thanks for bringing it up 😄